org.eclipse.cdt.cross.arc.gnu/src/org/eclipse/cdt/cross/arc/gnu/common/ARCGCCSpecsRunSIProvider.java

```diff
@@ -23,23 +23,25 @@
 
     @Override
     protected String[] setEnvironment(ICommandLauncher launcher, Properties initialEnv) {
+        CompilerPluginVersionsChecker.getCompilerPluginVersionsChecker()
+                .checkPluginVersions(collector);
+
         // Ensure that we have properties.
         Properties props = initialEnv != null ? initialEnv : launcher.getEnvironment();
-        
+
         // Get an absolute path to ../bin.
         String eclipsehome = Platform.getInstallLocation().getURL().getPath();
         File predefined_path_dir = new File(eclipsehome).getParentFile();
-        String predefined_path = predefined_path_dir + File.separator + "bin"+File.separator;
-        
+        String predefined_path = predefined_path_dir + File.separator + "bin" + File.separator;
```

Comment:
I don't like that patch that adds new feature is combined with code reformatting. Please extract this into a separate commit for just reformatting of this file.